All_home_works/hw2/task1.py

```diff
@@ -0,0 +1,89 @@
+"""
+Given a file containing text. Complete using only default collections:
+    1) Find 10 longest words consisting from largest amount of unique symbols
+    2) Find rarest symbol for document
+    3) Count every punctuation char
+    4) Count every non ascii char
+    5) Find most common non ascii char for document
+"""
+import string
+from typing import List
+
+PUNCTUATION = set(string.punctuation)
+CROP_PUNCTUATION = str.maketrans({char: "" for char in PUNCTUATION})
```

Comment:
`str.maketrans` can handle a different number of arguments.  Let's use it in another way.

Reply:
CROP_PUNCTUATION = str.maketrans('', '', string.punctuation)